### ISSUE TITLE
allow breaking up task's checklist items into separate tasks

### ISSUE THREAD

**Issue by mkae** [dmfs/opentasks#297]
Sometimes it might be useful to create individual tasks from a task's checklist

**Comment by alexander-alzate** [comment]
This could be achieved by using the iCalendar property "*REFERED-TO*" on child tasks/subtasks pointing to the parent "*UID*" property.

**Comment by jkufner** [comment]
Is there any other application that supports this?


Alexander Alzate wrote, on 3.3.2017 00:48:

**Comment by alexander-alzate** [comment]
Yes sir, there is a software called **GNOME ToDo**, in its new version they support subtask using this iCalendar property.

Please refer to [RFC5545](https://tools.ietf.org/html/rfc5545#page-56)

GNOME ToDo Patch:
Commit # [013e9e7b9c8dd2c88d1149b469977f958a29d5a5](https://git.gnome.org/browse/gnome-todo/commit/?id=013e9e7b9c8dd2c88d1149b469977f958a29d5a5)

I'm not sure if there is another application supporting this but this is _the way it is supposed to work_.

**Comment by alexander-alzate** [comment]
I'm so Sorry, I just realized that I was saying "REFERED-TO" but the actual property is "RELATED-TO".
